h/models/token.py

```diff
@@ -24,6 +24,9 @@ class Token(Base, mixins.Timestamps):
     #: to, for example, one of the short-lived JWTs that the client uses).
     prefix = u'6879-'
 
+    #: A prefix that identifies a token as a refresh token.
+    refresh_token_prefix = u'7980-'
```

Comment:
I was just checking with @nickstenning about why we have these prefixes, and he said they're just for humans to be able to recognize what type of token it is. In this case I don't think we need a prefix for the refresh tokens, as there will never be used anywhere else other than when refreshing an expired access token.